src/traces/histogram/attributes.js

```diff
@@ -71,14 +71,44 @@ module.exports = {
         ].join(' ')
     },
 
-    mode: {
+    cumulative: {
+        valType: 'boolean',
+        dflt: false,
+        role: 'info',
+        description: [
+            'If true, display the cumulative distribution by summing the',
+            'binned values. Use the `direction` and `centralbin` attributes',
+            'to tune the accumulation method.'
+        ].join(' ')
+    },
+
+    direction: {
         valType: 'enumerated',
-        values: ['density', 'cumulative'],
-        dflt: 'density',
+        values: ['increasing', 'decreasing'],
+        dflt: 'increasing',
         role: 'info',
         description: [
-            ''
-        ].join('')
+            'Only applies if `cumulative=true.',
+            'If *increasing* (default) we sum all prior bins, so the result',
+            'increases from left to right. If *decreasing* we sum later bins',
+            'so the fresult decreases from left to right.'
+        ].join(' ')
+    },
```

Comment:
I'm ok with `direction` here. But we should keep in mind that `direction` is now present in several attribute containers. Currently:

- In pie traces, `direction: 'clockwise' || 'counterclockwise'`
- In updatemenus, `direction: 'left' || 'right' || 'up' || 'down'`
- In the animtion config, `direction: 'forward' || 'reverse'`
- In base layout for polar plots: `direction: 'clockwise' || 'counterclockwise'`

I suppose it would be nice to make enumerated attributes of the same name share the same posibile values when used in different containers. Or maybe that's too much to ask for.